benchmarks/lstm_ip.json

```diff
@@ -0,0 +1 @@
+"[{\"input_sentence\": \"on the exchange floor as soon as ual stopped trading we <unk> for a panic said one top floor trader\"}]"
```

Comment:
Missing newline

Reply:
As discussed offline, this is not needed here.